src/navigators/AppStack.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { createStackNavigator } from '@react-navigation/stack';
+
+import { MAIN_SCREEN } from 'constants/screens';
+
+import MainScreen from 'screens/MainScreen';
+
+const Stack = createStackNavigator();
```

Comment:
Can you deconstruct here?
```suggestion
const { Navigator, Screen } = createStackNavigator();
```

Reply:
I am not so sure to do this. It might happen that we want to use a Tab navigation inside of a stack navigation and the Internal component names will Collide. Tab navigation also has Screen and Navigator. I think this verbosity is beneficial.

Reply:
Agree with maco here, I think it's a similar case to `Animated.View` where you want to have the prefix to avoid collisions